src/common/common.h

```diff
@@ -34,10 +56,11 @@
     backtrace_symbols_fd(buffer, calls, 1);                               \
     exit(-1);                                                             \
   } while (0);
+#endif
 
-#define CHECKM(COND, M, ...)                                 \
-  if (!(COND)) {                                             \
-    LOG_ERR("Check failure: %s \n" M, #COND, ##__VA_ARGS__); \
+#define CHECKM(COND, M, ...)                                   \
+  if (!(COND)) {                                               \
+    LOG_ERROR("Check failure: %s \n" M, #COND, ##__VA_ARGS__); \
```

Comment:
Hmm, I was just following standard logging levels found in other systems here. I think it makes sense to differentiate between LOG_WARN and LOG_ERROR. Here is a good post that talks about the different levels: [link](http://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels).

I'll look through the LOG_ERRORs and see which ones might need to be LOG_FATAL.